google-account-plugin/build.gradle

```diff
@@ -1,84 +1,36 @@
-// see https://github.com/JetBrains/gradle-intellij-plugin
+plugins {
+    id "org.jetbrains.intellij" version "0.0.29"
```

Comment:
Are we gonna use 28 or 29 across plugins? Can we do this in the root project and apply it in the subprojects {} closure so we don't have to keep this in 4 different files